macros/models/_sector/tokens/transfers_enrich.sql

```diff
@@ -75,11 +77,22 @@ LEFT JOIN
     {{ tokens_erc20_model }} as tokens_erc20
     ON tokens_erc20.blockchain = t.blockchain
     AND tokens_erc20.contract_address = t.contract_address
-LEFT JOIN prices
-    ON prices.blockchain = '{{ blockchain }}' -- due to data volume in prices, filter by static blockchain value for performance gains (slight improvement over using t.blockchain directly)
-    AND (
-            prices.contract_address = t.contract_address
-            OR (t.contract_address IS NULL AND prices.contract_address = evms_info.wrapped_native_token_address)
-        )
-    AND prices.minute = date_trunc('minute', t.block_time)
+LEFT JOIN
+    prices
+    ON date_trunc('minute', t.block_time) = prices.minute
+    AND CASE
+        WHEN t.token_standard = 'native'
+        THEN prices.blockchain IS NULL
```

Comment:
Can't these be one statement by using `AND` in between, similar to this https://github.com/duneanalytics/spellbook/blob/e89871de0a0e6a719382c32bdce5b01cf70e94d2/macros/models/_sector/tokens/balances_enrich.sql#L27-L31

Reply:
pushed a change to reformat the case statement. i don't mind the approach either way